rs/bitcoin/ckbtc/minter/src/test_fixtures.rs

```diff
@@ -142,3 +142,309 @@ pub mod mock {
         }
     }
 }
+
+pub mod arbitrary {
+    use crate::{
+        address::BitcoinAddress,
+        signature::EncodedSignature,
+        state::{
+            eventlog::{Event, EventType},
+            ChangeOutput, Mode, ReimbursementReason, RetrieveBtcRequest, SuspendedReason,
+        },
+        tx,
+        tx::{SignedInput, TxOut, UnsignedInput},
+    };
+    use candid::Principal;
+    pub use event::event_type;
+    use ic_base_types::CanisterId;
+    use ic_btc_interface::{OutPoint, Satoshi, Txid, Utxo};
+    use icrc_ledger_types::icrc1::account::Account;
+    use proptest::{
+        array::uniform20,
+        array::uniform32,
+        collection::{vec as pvec, SizeRange},
+        option,
+        prelude::{any, Just, Strategy},
+        prop_oneof,
+    };
+    use serde_bytes::ByteBuf;
+
+    // Macro to simplify writing strategies that generate structs.
+    macro_rules! prop_struct {
+        ($struct_path:path { $($field_name:ident: $strategy:expr),* $(,)? }) => {
+            #[allow(unused_parens)]
+            ($($strategy),*).prop_map(|($($field_name),*)| {
+                $struct_path {
+                    $($field_name),*
+                }
+            })
+        };
+    }
+
+    fn amount() -> impl Strategy<Value = Satoshi> {
+        1..10_000_000_000u64
+    }
+
+    fn txid() -> impl Strategy<Value = Txid> {
+        pvec(any::<u8>(), 32).prop_map(|bytes| {
```

Comment:
nit: With`uniform32(any::<u8>())` you directly obtain an array of the right size.

Reply:
Good point, thanks for the suggestion!